petroflow/src/well.py

```diff
@@ -367,3 +368,137 @@ def drop_short_segments(self, min_length):
         for well in wells:
             well.segments = [segment for segment in well if segment.length > min_length]
         return self.prune()
+
+    def _aggregate_array(self, func, attr, aggregate):
+        """Aggregate or concatenate loaded attributes
+        from `WellSegment.attrs_pixel_index`.
+
+        Parameters
+        ----------
+        func : {'mean', 'max'}
+            Name of aggregation function. Now there is only `mean` and 'max'!
+        attr : str
+            Name of attribute.
+        aggregate: bool
+            If `True`, values will be aggregate.
+            If `False`, values will be concatenate.
+
+        Returns
+        -------
+        numpy.ndarray
+            Assembled array.
+        """
+        func = 'mean' if func != 'max' else 'max'
+        if getattr(self.segments[0], '_'+attr) is None:
+            return None
+        pixels_per_m = self.segments[0].pixels_per_cm * 100
+
+        agg_array_hight_pix = round((self.depth_to-self.depth_from)*pixels_per_m)
```

Comment:
```suggestion
        agg_array_hight_pix = round((self.depth_to - self.depth_from) * pixels_per_m)
```
Also `hight` -> `height` hereinafter.